uavcan_data_types/cvra/io/20131.ServoPWM.uavcan

```diff
@@ -6,3 +6,5 @@
 uint7 node_id
 
 float16[4] servo_pos
+float16[4] servo_vel
```

Comment:
Maybe document the fact that negative values are interpreted as "Do not change".